### PR TITLE
Move tolerance option

### LINKED ISSUE #6831: Option for adjusting move distance tolerance
If a users pan the map when clicking the click will be parsed as a move instead of a click. A group of our mobile users have problem with clicking on the map without triggering a move. Our suggested solution is to make the `moveTolerance` variable in `ol.MapBrowserEventHandler` configurable via a `moveTolerance` option in `ol.Map`.

Proposed implementation:
https://github.com/notnotse/openlayers/tree/move-tolerance-option

### PR DESCRIPTION
**Pull request by itjope**
Make the `moveTolerance` variable in `ol.MapBrowserEventHandler` configurable via a `moveTolerance` option in `ol.Map`

Fix #6831

### CODE REVIEW

**Comment by bjornharrtell** [comment]
Nice work @itjope, I think would be useful to have this in.

**Comment by ahocevar** [comment]
Thanks for this contribution @itjope. Can you please submit a [CLA](https://github.com/openlayers/cla/) so we can merge this?

File: externs/olx.js [hunk 1]
```diff
@@ -386,6 +387,17 @@ olx.MapOptions.prototype.logo;
 
 
 /**
+ * The minimum distance in pixels the cursor must move to be detected
+ * as a map move event instead of a click. Increasing this value can make it
+ * easier to click on the map.
+ * Default is `1`.
```
Comment (fredj) [inline externs/olx.js]: Default is `window.devicePixelRatio`.

Comment (ahocevar) [inline externs/olx.js]: > Default is window.devicePixelRatio.
Depends on how we define 'pixel'. Elsewhere, e.g. for `ol.Map#getSize`, we use JS pixels. So I'd prefer to leave this as `1`.

Comment (ahocevar) [inline externs/olx.js]: Actually the default is `1`, because the value given here is multiplied with the device pixel ratio.

Comment (fredj) [inline externs/olx.js]: you're right, the default is `1`, sorry for the noise

**Comment by itjope** [comment]
@ahocevar  I have now sent a scanned signed copy of the CCLA.

**Comment by TheNaderio** [comment]
Is this tolerance available in OpenLayers.Map()? (2.13)

**Pull request merged**